crowdsourcing/models.py

```diff
@@ -559,3 +559,36 @@ class TaskComment(models.Model):
     task = models.ForeignKey(Task, related_name='taskcomment_task')
     comment = models.ForeignKey(Comment, related_name='taskcomment_comment')
     deleted = models.BooleanField(default=False)
+
+
+class FinancialAccount(models.Model):
+    owner = models.ForeignKey(UserProfile, related_name='account_owner', null=True)
```

Comment:
related_name should be named as plural of Class for ForeignKey - In this case, it should be financial_accounts. So it will eventually be used as owner.financial_accounts. It also gives indication that it is 1 to many connection while coding.